verifier.js

```diff
@@ -52,7 +52,17 @@ const PM = exports.PM = new ProcessManager({
 		} catch (e) {}
 
 		return success ? 1 : 0;
-	},
+	}
+}
+
+exports.VerifierManager = VerifierManager;
+
+const VERIFIER_PROCESSES = 1;
```

Comment:
...yes. I misunderstood what those last few comments were meant to be tellingg me what to do. Sounds like I _missinterpreted_ the point, but made a pun from the past instead